aeron-cluster/src/main/java/io/aeron/cluster/ClusterBackupAgent.java

```diff
@@ -578,7 +614,7 @@ else if (NULL_VALUE == correlationId && consensusPublication.isConnected())
                 ctx.consensusStreamId(),
                 AeronCluster.Configuration.PROTOCOL_SEMANTIC_VERSION,
                 ctx.consensusChannel(),
-                ArrayUtil.EMPTY_BYTE_ARRAY))
+                ctx.credentialsSupplier().encodedCredentials()))
```

Comment:
Only if the backup query never receives a response and the implementation of the supplier allocates.  Otherwise it will be once an hour if the agent successfully starts backing up or once every 5 seconds if the backup operation fails elsewhere (unless I've misread the code).